ramani-maplibre/src/main/java/org/ramani/compose/MapLibre.kt

```diff
@@ -169,12 +169,12 @@ fun MapLibre(
 
         maplibreMap.addOnMapClickListener { latLng ->
             onMapClick(latLng)
-            true
+            false
```

Comment:
I will have to make a few tests to make sure that this doesn't break something else. But I can't think of such an issue off the top of my head :+1:, hopefully it is just fine :blush: